examples/nlp/duplex_text_normalization/analyze_errors.py

```diff
@@ -284,8 +285,8 @@ def analyze(errors_log_fp: str, visualization_fp: str):
 if __name__ == '__main__':
     # Parse argument
     parser = ArgumentParser()
-    parser.add_argument('--errors_log_fp', type='str', help='Path to the error log file', required=True)
-    parser.add_argument('--visualization_fp', type='str', help='Path to the output visualization file', required=True)
+    parser.add_argument('--errors_log_fp', help='Path to the error log file', required=True)
+    parser.add_argument('--visualization_fp', help='Path to the output visualization file', required=True)
```

Comment:
I still get error:
  File "analyze_errors.py", line 160, in from_lines
    return cls(_input, target, pred, mode)
UnboundLocalError: local variable 'pred' referenced before assignment